harnessed_jobs/rtm_aliveness_power_on/v0/ccs_rtm_aliveness_power_on.py

```diff
@@ -94,16 +94,9 @@ def reb_power_on(ccs_sub, rebid, power_line, ccd_type, raise_exception=True):
 
     logger.info("Turn on REB clock and rail voltages.")
 
-    # Load sensor-specific configurations.
-    if ccd_type == 'ITL':
-        ccs_sub.ts8.synchCommand(10, "loadCategories Rafts:itl")
-        ccs_sub.ts8.synchCommand(10, "loadCategories RaftsLimits:itl")
-    elif ccd_type.upper() == 'E2V':
-        ccs_sub.ts8.synchCommand(10, "loadCategories Rafts:e2v")
-        ccs_sub.ts8.synchCommand(10, "loadCategories RaftsLimits:e2v")
-    else:
-        raise java.lang.Exception("ccs_rtm_aliveness_power_on: Invalid ccd_type, "
-                                  + ccd_type)
+    # Load configurations
+    ccs_sub.ts8.synchCommand(10, "loadCategories Rafts")
+    ccs_sub.ts8.synchCommand(10, "loadCategories RaftsLimits")
```

Comment:
@slm23 It wasn't clear from your description whether these lines should be modified like this or simply omitted entirely.

Reply:
@slm23 or @ohsaxton Could you comment please on what needs to be changed here?  I'd like to get this and other changes into prod asap.